### DISCUSSION TITLE
Libellée dans le json de sortie

### DISCUSSION THREAD

**Discussion by ChrisThe01** [MTESSDev/FRW#143] [Annonces]
Bonjour,

Comment fait-on pour avoir les libellé des champs dans le json de sortie lorsqu'on fait soumettre ?

Merci

**Comment by anisite** [comment]
Bonsoir, 

il suffit normalement d'ajouter l'étape de tâche ``extraireQuestions`` dans votre fichier de transmission tel que documenté ici :

[fichiers-transmission.md#exemple-de-toutes-les-possibilit%C3%A9es-de-configuration](https://github.com/MTESSDev/FRW/blob/main/Documentation/fichiers-transmission.md#exemple-de-toutes-les-possibilit%C3%A9es-de-configuration)

les informations seront alors automatiquement ajoutées.

Merci
